frame/clearing-house/src/lib.rs

```diff
@@ -1111,11 +1120,22 @@ pub mod pallet {
 			// Settle funding payments and outstanding profits for all positions in the account
 			let mut positions = Self::get_positions(&account_id);
 			for position in positions.iter_mut() {
-				Self::settle_funding_and_outstanding_profits(
-					account_id,
-					position,
-					&mut collateral,
-				)?;
+				let market = Self::try_get_market(&position.market_id)?;
+				Self::settle_funding(position, &market, &mut collateral)?;
+			}
+
+			let mut available_profits = Self::available_profits().unwrap_or_else(Zero::zero);
+			if !available_profits.is_zero() {
+				let mut outstanding_profits =
+					Self::outstanding_profits(account_id).unwrap_or_else(Zero::zero);
+
+				let realizable_profits = cmp::min(outstanding_profits, available_profits);
```

Comment:
Any reason not to use `outstanding_profits.min(available_profits)` over `cmp::min(outstanding_profits, available_profits)`?

I see you're using the `min` method from the `sp_std` crate, but wouldn't this method be used in either syntax anyway?

Reply:
I think I just forgot to update that part to use the trait method. Thanks for pointing it out!